airbyte-cdk/python/airbyte_cdk/sources/file_based/schema_validation_policies/default_schema_validation_policies.py

```diff
@@ -4,30 +4,34 @@
 
 from typing import Any, Mapping, Optional
 
-from airbyte_cdk.sources.file_based.exceptions import FileBasedSourceError, StopSyncPerValidationPolicy
+from airbyte_cdk.sources.file_based.exceptions import FileBasedSourceError, RecordParseError, StopSyncPerValidationPolicy
 from airbyte_cdk.sources.file_based.schema_helpers import conforms_to_schema
 from airbyte_cdk.sources.file_based.schema_validation_policies import AbstractSchemaValidationPolicy
 
 
 class EmitRecordPolicy(AbstractSchemaValidationPolicy):
     name = "emit_record"
 
-    def record_passes_validation_policy(self, record: Mapping[str, Any], schema: Optional[Mapping[str, Any]]) -> bool:
+    def record_passes_validation_policy(self, record: Optional[Mapping[str, Any]], schema: Optional[Mapping[str, Any]]) -> bool:
+        if record is None:
+            raise RecordParseError(FileBasedSourceError.ERROR_PARSING_RECORD)
         return True
 
 
 class SkipRecordPolicy(AbstractSchemaValidationPolicy):
     name = "skip_record"
 
-    def record_passes_validation_policy(self, record: Mapping[str, Any], schema: Optional[Mapping[str, Any]]) -> bool:
-        return schema is not None and conforms_to_schema(record, schema)
+    def record_passes_validation_policy(self, record: Optional[Mapping[str, Any]], schema: Optional[Mapping[str, Any]]) -> bool:
+        return schema is not None and record is not None and conforms_to_schema(record, schema)
```

Comment:
Even with the `skip_record` policy, I think that a `None` record would be invalid, if it's indicative of an issue with parsing the file.